src/components/fields/text-field/text-field.mcss

```diff
@@ -10,3 +10,8 @@
 .code {
   composes: mono from styles;
 }
+
+:disabled {
```

Comment:
This isn’t quite the right way to do this, @dNitza. CSS Modules stuff should be local and I think that selector will end up global by default. I’m going to merge the `emotion` branch into master soon (today hopefully) so perhaps back this out and we can redo it with the emotion setup?